Sources/NIO/DeadChannel.swift

```diff
@@ -12,10 +12,14 @@
 //
 //===----------------------------------------------------------------------===//
 
+import Baggage
+
 /// A `DeadChannelCore` is a `ChannelCore` for a `DeadChannel`. A `DeadChannel` is used as a replacement `Channel` when
 /// the original `Channel` is closed. Given that the original `Channel` is closed the `DeadChannelCore` should fail
 /// all operations.
 private final class DeadChannelCore: ChannelCore {
+    var baggage = BaggageContext()
```

Comment:
not sure about usage of DeadChannelCore, but sounds like it may be safer to never store baggage values in here, there's no real purpose to propagate using a dead channel it feels like?